components/bootloader_support/src/esp32/bootloader_esp32.c

```diff
@@ -211,6 +211,13 @@ esp_err_t bootloader_init(void)
     // print 2nd bootloader banner
     bootloader_print_banner();
 
+#ifdef CONFIG_ESP_SIMPLE_BOOT
```

Comment:
I'm guessing you're just keeping in sync with PR #336

Reply:
Correct. Well, that one was kept in sync with this.